modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java

```diff
@@ -1205,9 +1207,11 @@ private FileEntry _updateFileEntry(
 			User user = _userLocalService.getUser(themeDisplay.getUserId());
 
 			Date expirationDate = _getExpirationDate(
-				uploadPortletRequest, user.getTimeZone());
+				uploadPortletRequest, cmd.equals(Constants.ADD_DYNAMIC),
```

Comment:
Conditional expressions like `cmd.equals(...)` are usually avoided; you need to use a conditional statement:
```
boolean addDynamic = false;

if (cmd.equals(...)) {
   addDynamic = true;
}
```
and then use the variable.

Reply:
ok